tests.py

```diff
@@ -601,7 +601,7 @@ def run_tests():
     log('')
 
     success, payload = _execute_test(
-        'admin/get_user_posts.json',
+        'admin/get_client_posts.json',
```

Comment:
You will when you implement it, however it did not look like you had done anything with the client posts yet.  This would be an end point that would be called to get posts just from a specific user.